x/ibc/handler.go

```diff
@@ -192,8 +192,12 @@ func NewHandler(k keeper.Keeper) sdk.Handler {
 				return nil, sdkerrors.Wrap(err, "receive packet callback failed")
 			}
 
+			if err := k.ChannelKeeper.ReceiveExecuted(ctx, cap, msg.Packet); err != nil {
```

Comment:
maybe we should rename this to `WriteReceipt`?

Reply:
ReceiveExecuted just increases the sequence or writes an empty receipt